File: despatch.WEB/app/src/swagger/models/contact-data-urls-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ContactDataResponse } from './contact-data-response';
import { UrlResponse } from './url-response';
export interface ContactDataUrlsResponse extends BaseEntityResponse{
  contactData?: ContactDataResponse;
  contactDataId?: string;
  url?: UrlResponse;
  urlId?: string;
}

File: despatch.Domain/Models/Default/Response/ContactDataUrlsResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class ContactDataUrlsResponse : BaseEntityResponse
    {
        public string UrlId { get; set; }
        public string ContactDataId { get; set; }

        public virtual UrlResponse Url { get; set; }
        public virtual ContactDataResponse ContactData { get; set; }
    }
}
